recipes/GillesPy2/LICENSE

```diff
@@ -0,0 +1,674 @@
+GNU GENERAL PUBLIC LICENSE
```

Comment:
This is fine for now but the licence file should be included in the packages `MANIFEST.in` so they sdist on PyPI is properly compliant with the license.